modules/nodes-nodes-audit-log-basic-viewing.adoc

```diff
@@ -32,7 +32,7 @@ ip-10-0-170-165.ec2.internal audit.log
 .. View the {product-title} API server log for a specific master node and timestamp or view all the logs for that master:
 +
 ----
-$ oc adm node-logs <node-ip> <log-name> --path=openshift-apiserver/<log-name>
+$ oc adm node-logs <node-ip> --path=openshift-apiserver/<log-name>
```

Comment:
@soltysh You remove <log-name> here, but the two example below still provide that argument:
* audit-2019-04-08T13-09-01.227.log
* audit.log

Should those be removed from the examples?

Reply:
Good catch - fixed.